UnityProject/Assets/Scripts/Tilemaps/Behaviours/Layers/MetaDataLayer.cs

```diff
@@ -371,6 +363,14 @@ public void PaintBlood(Vector3 worldPos, ReagentMix reagents)
 		BloodDry(worldPos.RoundToInt());
 	}
 
+	public void CreateLiquidOverlay(Vector3Int localPosInt, ReagentMix reagents)
+	{
+		var liquidColor = reagents.MixColor;
+		liquidColor.a = Mathf.Clamp(liquidColor.a, 0, 0.65f); //makes sure liquids don't completely hide everything behind it.
+		matrix.MetaTileMap.AddOverlay(localPosInt, TileType.UnderObjectsEffects, "BaseLiquid", color: liquidColor);
```

Comment:
I would say wouldn't use string?> but idk

Reply:
have you seen half of this script? all of it needs to stop using strings